### PR TITLE
ENT-3095 | Added courses in program search endpoint.

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -2175,11 +2175,25 @@ class Meta:
...
+        return MinimalCourseSerializer(
+            result.object.courses.all(),
+            many=True,
+            context={
+                'request': self.context.get('request'),
+            }
+        ).data
+
+    def get_applicable_seat_types(self, result):
+        return list(result.object.type.applicable_seat_types.values_list('slug', flat=True))
```
Comment (saleem-latif) [inline course_discovery/apps/api/serializers.py:2195]: Do we need to handle `AttributeError` here? or that will never happen?

**Review by saleem-latif — APPROVED** [review]
One comment, otherwise Looks good 👍

**Review by Dillon-Dumesnil — APPROVED** [review]

**Review by jlajoie — APPROVED** [review]
It doesn't let me remove myself as a reviewer and I'm cleaning up my requests in GitHub, so sure 👍🏼

**Comment by DawoudSheraz** [comment]
I am assuming this is no longer needed since there has been no update on it. Closing this.

**Comment by openedx-webhooks** [comment]
Although this pull request is already closed,  I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks.

**Pull request closed without merging**